source/application/AVIDumping/WaitableBool.cpp

```diff
@@ -6,52 +6,51 @@
 
 #include "WaitableBool.h"
 
-WaitableBool::WaitableBool(bool initialState)
-	: m_value(initialState)
+WaitableBool::WaitableBool(bool initialState) : m_value(initialState)
```

Comment:
Can we always linebreak on this case for consistency? (new line before `:`)

Reply:
Doesn't look like it.